puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-pipeline.groovy

```diff
@@ -13,7 +13,9 @@ pipeline {
             agent { label 'sshkey' }
             steps {
                 mash("foreman-mash-split.py", foreman_version)
-                mash("foreman-rails-mash-split.py", foreman_version)
+                if(foreman_version != '2.0') { 
```

Comment:
I think this wold be more robust if it checked for `1.23` and `1.24` explicitly. Otherwise, when 2.1 rolls around this will break.